jenkins/opensearch/benchmark-pull-request.jenkinsfile

```diff
@@ -278,9 +279,29 @@ pipeline {
                       """
                       script {
                         if (currentBuild.rawBuild.getCauses().toString().contains("GenericCause")) {
-                            withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
-                                def pull_request = Integer.parseInt("${pull_request_number}")
-                                sh ("gh pr comment ${pull_request} --repo ${repository} --body-file final_result_${BUILD_NUMBER}.md")
+                            if ("${pull_request_number}" != '' && "${repository}" != '') {
+                                withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
+                                    def pull_request = Integer.parseInt("${pull_request_number}")
+                                    sh ("gh pr comment ${pull_request} --repo ${repository} --body-file final_result_${BUILD_NUMBER}.md")
+                                }
+                                if ("${baseline_cluster_config}" != '') {
+                                        Map <String, String> benchmarkTestIds = getCompareBenchmarkIds(
+                                                                                   baselineClusterConfig: "${baseline_cluster_config}",
+                                                                                   distributionVersion: "${DISTRIBUTION_VERSION}-SNAPSHOT",
+                                                                                   workload: "${TEST_WORKLOAD}",
+                                                                                   pullRequestNumber: "${pull_request_number}"
+                                                                                )
+                                        // Trigger compare-benchmarks job
+                                        build job: 'compare-benchmarks',
```

Comment:
Nit: Possible of changing this to `benchmark-compare` to in sync with other benchmark workflow naming convention?

Thanks.